src/main/index.ts

```diff
@@ -123,6 +124,37 @@ export async function readThriftFiles(options: {
     return thriftFiles
 }
 
+function parseIncludes(parsedFiles: Array<IParsedFile>, options: IMakeOptions): Array<IParsedFile> {
+    const parsedFilesIncludes: IFileIncludes = parsedFiles.reduce(
+        (acc: IFileIncludes, parsedFile, index) => {
+            const namedIncludes: IFileIncludes = {};
+            Object.keys(parsedFile.includes).forEach(include => {
+                const includePath = path.resolve(parsedFile.includes[include].importedFrom, parsedFile.includes[include].path);
```

Comment:
Would like to give a little flexibility to the way we import idls from either a relative path or absolute path from the sourceDir.
```
                let includePath = path.resolve(
                    parsedFile.includes[include].importedFrom,
                    parsedFile.includes[include].path,
                )
                if (!fs.existsSync(includePath)) {
                    includePath = path.resolve(
                        options.sourceDir,
                        parsedFile.includes[include].path,
                    )
                }
```